tls/s2n_kem.c

```diff
@@ -30,9 +32,15 @@ const struct s2n_kem s2n_kyber_512_r3 = {
         .private_key_length = S2N_KYBER_512_R3_SECRET_KEY_BYTES,
         .shared_secret_key_length = S2N_KYBER_512_R3_SHARED_SECRET_BYTES,
         .ciphertext_length = S2N_KYBER_512_R3_CIPHERTEXT_BYTES,
+#if defined (AWS_LC_SUPPORTS_KYBER_512)
+        .generate_keypair = &s2n_kyber_evp_generate_keypair,
+        .encapsulate = &s2n_kyber_evp_encapsulate,
+        .decapsulate = &s2n_kyber_evp_decapsulate,
```

Comment:
Naming nitpick, these function names should be updated to `s2n_kyber_512_evp_*` since we'll likely be adding Kyber_768 soon.

Reply:
ack